src/github/issuesParser.ts

```diff
@@ -0,0 +1,33 @@
+import { debug } from "@actions/core";
+import { GitHub } from "@actions/github/lib/utils";
+import moment from "moment";
+
+export interface IssueData {
+    html_url: string;
+    title: string;
+    created_at: string;
+    updated_at: string;
+    number: number;
+}
+
+interface Repo {
+    owner: string,
+    repo: string;
+}
+
+export const fetchIssues = async (octokit: InstanceType<typeof GitHub>, daysStale: number, repo: Repo): Promise<IssueData[]> => {
+    const issues = await octokit.rest.issues.listForRepo({ ...repo, per_page: 100, state: "open" });
```

Comment:
makes sense to add pagination
as at least this one https://github.com/paritytech/ink has > 100

may be as separate ticket would also be good to add filter by label.. so in such repos like substrate you could filter out the particular team

Reply:
Good catch, I'll add a pagination system in a different commit

Reply:
Created #4 to handle it